test-v13/src/test/java/permissions/dispatcher/test_v13/FragmentWithAllAnnotationsKtPermissionsDispatcherTest.kt

```diff
@@ -0,0 +1,149 @@
+package permissions.dispatcher.test_v13
+
+
+import android.support.v4.app.ActivityCompat
+import org.junit.Before
+import org.junit.BeforeClass
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Matchers.any
+import org.mockito.Mockito
+import org.powermock.api.mockito.PowerMockito
+import org.powermock.core.classloader.annotations.PowerMockIgnore
+import org.powermock.core.classloader.annotations.PrepareForTest
+import org.robolectric.RobolectricTestRunner
+import permissions.dispatcher.PermissionRequest
+import permissions.dispatcher.PermissionUtils
+import org.powermock.modules.junit4.rule.PowerMockRule
+import org.junit.Rule
+import permissions.dispatcher.test.*
+
+
+@Suppress("IllegalIdentifier")
+@RunWith(RobolectricTestRunner::class)
+@PowerMockIgnore("org.mockito.*", "org.robolectric.*", "android.*")
+@PrepareForTest(PermissionUtils::class)
```

Comment:
All test cases mock `PermissionUtils` class but I start feeling this should not be mock...let me change where to mock. I think `PermissionChecker` should be the class to mock